logback/src/main/java/com/grafana/loki/HttpHandler.java

```diff
@@ -0,0 +1,60 @@
+package com.grafana.loki;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpRequest.Builder;
+import java.net.http.HttpResponse;
+
+public class HttpHandler {
+	private static HttpClient client;
+
+	public static void sendLoki(String tenantID, URI url, byte[] body) {
+		int retryCount = 3;
+		int retryDelay = 2000;
```

Comment:
I think we would like these to be configurable so people can choose their own timeout

Reply:
Yeah, this was just for testing. Will make this as configurable param